llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -452,8 +452,20 @@ std::string saveModuleProperties(module_split::ModuleDesc &MD,
   if (MD.isESIMD()) {
     PropSet[PropSetRegTy::SYCL_MISC_PROP].insert({"isEsimdImage", true});
   }
-  if (MD.isLargeGRF())
-    PropSet[PropSetRegTy::SYCL_MISC_PROP].insert({"isLargeGRF", true});
+
+  {
+    // check for large GRF property
+    bool HasLargeGRF = false;
+    for (const auto *F : MD.entries()) {
+      if (F->hasFnAttribute(::sycl::kernel_props::ATTR_LARGE_GRF)) {
+        HasLargeGRF = true;
+        break;
+      }
+    }
```

Comment:
I suggest using `std::find_if` instead of manually written loop.

Reply:
Switched to `llvm::any_of` in aaf1e92b82415656faec197d9afa340fe54627bd